Simulator/Simulator/Simulator/src/Neuron.c

```diff
@@ -228,6 +255,31 @@ ERROR
 }
 
 
+String* neuron_class_get_desc(NeuronClass* neuron_class) {
```

Comment:
Don't thing this is worth testing, is to simple, add a comment to specify that is not tested

Reply:
Added 'untested' comment for it